src/Symfony/Component/DependencyInjection/Loader/YamlFileLoader.php

```diff
@@ -86,13 +88,15 @@ class YamlFileLoader extends FileLoader
         'calls' => 'calls',
         'tags' => 'tags',
         'autowire' => 'autowire',
+        'autoconfigure' => 'autoconfigure',
```

Comment:
Ha, so now we've reversed roles :).

Maybe? I have an IntegrationTest testing the override logic for it, so that's solid. In theory, I could use `autoconfigure: true` under `_defaults`, but then set `autoconfigure: false` for one specific interface (e.g. `VoterInterface`) because I'm doing something weird and want to control all of my voters entirely myself.